tools/clang/unittests/HLSLExec/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ set_target_properties(ExecHLSLTests PROPERTIES FOLDER "Execution tests")
 
 target_link_libraries(ExecHLSLTests PRIVATE
   dxcompiler
+  LLVMDxcSupport
```

Comment:
If we can get rid of the DxcCoGetMalloc dependency (see other comment), can we get rid of this dependency as well?

Reply:
Yes, and done.